nucypher/blockchain/eth/token.py

```diff
@@ -404,8 +445,10 @@ def divide(self, target_value: NU, additional_periods: int = None) -> Tuple['Sta
         self.sync()
 
         # Ensure selected stake is active
-        if self.is_expired:
-            raise self.StakingError(f'Cannot divide an expired stake. Selected stake expired {self.unlock_datetime}.')
+        status = self.status()
+        if not status.is_child(Stake.Status.DIVISIBLE):
```

Comment:
Maybe you can create a `is_divisible` property in `Status`. Same for `is_editable`.

Reply:
It was there but `Status` is for removing these methods, now we can describe sub-stake just with one status variable, otherwise will be needed to create flags for searching like it was before, and for each search - we need aditional flag. So I see that `Status` reduces code